api/app/tox.ini

```diff
@@ -14,7 +14,7 @@ whitelist_externals =
     flake8
     isort
 commands =
-    pytest: pytest --cov=signals --cov-report term --no-cov-on-fail {posargs:} --tb=short
+    pytest: pytest -n 4 --cov=signals --cov-report term --no-cov-on-fail {posargs:} --tb=short
```

Comment:
We had something like this before. I think we accidentally removed it in one of our previous commits 😄 
We had it like: 
```
[pytest]
addopts = -n3
```
Is it sill possible to disable this behaviour this way? By running the tests like: `docker-compose run --rm api tox -- -n0`?

Reply:
I did wonder why the pytest-xdist was included but not used...
And yes it possible to override with n0 or n1.

Reply:
The reason was that for running the tests locally and dropping into PDB or showing standard out you cannot run in parallel. But the -n0 switch could override this behavior when running locally.

Reply:
Yeah whenever it fails, override n0 for debug.